+++
# Custom widget.
# An example of using the custom widget to create your own homepage section.
# To create more sections, duplicate this file and edit the values below as desired.
widget = "custom"
active = true
date = 2016-04-20T00:00:00

# Note: a full width section format can be enabled by commenting out the `title` and `subtitle` with a `#`.
title = "Research"
subtitle = "Diversity and oceanic distribution of marine phytoplankton"


# Order that this section will appear in.
weight = 4

+++

I believe an interdisciplinary, open and collaborative science can only bring solid knowledge and a better understanding of the world we live and share with the microbes.  

## **Diversity and oceanic distribution of marine phytoplankton** 

Changes in marine plankton communities driven by environmental variability ultimately impact the marine food web and global biogeochemical cycles (pathway by which a chemical substance moves through biotic and abiotic compartments of Earth). In order to predict and assess community shifts and their consequences, we first need to determine the composition of these communities and how they change along environmental and biological gradients, for example latitude or temperature. 

With the advent of molecular biology methods such as high throughput sequencing of gene markers (metabarcoding) and an increasing number of sequence datasets, access to the global scale distribution of your favorite “bug” became more feasible. Basically, after acquiring sequence data either in the lab or through public databases, all you need is to determine which sequence belongs to which bug, pretty much like sorting the sequences into boxes where each box represents different individuals. The challenge here is to determine the individual units (like deciding the size of the box) and their close relatives, so you don’t end up sorting the sequences into the wrong compartment. 

One common language we all use in biology to define the limits between individuals is the one provided by taxonomy, where the individuals are categorized into species, the different species into genera, etc... The combination of taxonomy and metabarcoding method brings fascinating insights into marine phytoplankton distribution and ecology. 

Below you can see examples of how we combined taxonomic knowledge with metabarcoding data to unveil the distribution of phytoplankton groups. 
 

## Bolidophyceae, the sister group of diatoms
![](/img/Bolido1.png) 
([Ichinomiya and Lopes dos Santos et al.,2016](files/papers/Ichinomiya_2016.pdf))

Late in the 1990’s, naked flagellated cells (previously called Bolidomonas, A) were isolated and described as belonging to the class Bolidophyceae. The silicified Parmales group (B) was discovered and observed in marine waters back in the 1970’s but their taxonomic affiliation was an enigma until the isolation of one Parmales strain in 2011 when by DNA sequence identity Bolidophyceae and Parmales were thought to be sister classes or even organisms within the same class. In our study ([Ichinomiya and Lopes dos Santos et al., 2016](files/papers/Ichi nomiya_2016.pdf)), we acquired sequence data from strains of both groups to analyze the evolutionary relationships among several silicified and flagellated cells isolated from different oceanic regions. We were able to confirm that the silica covered cells belong to the same Class (Bolidophyceae) than the naked flagellated cells, suggesting that Bolidophyceae may have a life cycle with motile flagellated and non-motile silicified stages. Our data allowed to determine the distribution of the major Bolidophyceae taxa across the world oceans using metabarcoding data set obtained in the frame of the Tara Oceans expedition. We observed that silicified cells are mostly found in cold-water while the naked flagellate have temperate and tropical distributions. 
Many questions about Bolidophyceae remain open, so in collaboration with [Akira Kuwata]( https://www.researchgate.net/profile/Akira_Kuwata) we plan to continue exploring the diversity of this groups and the ecological factors that determine its growth and differentiation into flagellate and silicified cells.  

## Chloropicophyceae also known as Prasinophytes clade VII 
![](/img/chloropico.png)
([Lopes dos Santos et al., 2017](files/papers/ Lopes_2017b.pdf))

Chloropicophyceae is a group of pico (2 – 3 µm) planktonic green algae that were known previously as prasinophytes clade VII ([Lopes dos Santos et al., 2017](files/papers/ Lopes_2017b.pdf)). Since the isolation of the first strain in 1965, numerous rRNA sequences belonging to this group have been retrieved from the marine environment and many strains isolated. We gathered the sequence data available from strains and environmental samples to analyze the diversity of these tiny little green balls. First, by using different gene markers, we demonstrated that this group is composed of at least 10 different clades: A1 to A7 and B1 to B3 ([Lopes dos Santos et al., 2017](files/papers/L opes_2017a.pdf)) which today is represented by at least one described species.  By using metabarcoding data from Tara Oceans and Ocean Sampling Day projects we were able to show that this group is one of the dominant Chlorophyta (green algae) group in oceanic waters, especially in areas where nutrients concentrations are moderately low. Currently, in collaboration with [Claude Lemieux] (https://www.researchgate.net/profile/Claude_Lemieux2), from Laval University, we are gathering information from Chloroplast and Mitochondrial genomes of Chloropicophyceae species to access the relationship among them and their environmental adaptation.  






