+++
# Custom widget.
# An example of using the custom widget to create your own homepage section.
# To create more sections, duplicate this file and edit the values below as desired.
widget = "custom"
active = true
date = 2016-04-20T00:00:00

# Note: a full width section format can be enabled by commenting out the `title` and `subtitle` with a `#`.
title = "Research"
subtitle = "Diversity and oceanic distribution of marine phytoplankton"


# Order that this section will appear in.
weight = 9

+++


Changes in marine plankton communities driven by environmental variability ultimately impact the marine food web and global biogeochemical cycles (pathway by which a chemical substance moves through biotic and abiotic compartments of Earth). To predict and assess community shifts and their consequences, we first need to determine the composition of these communities and how they change along environmental and biological gradients, for example latitude or temperature. 

With the advent of molecular biology methods such as high throughput sequencing of gene markers (metabarcoding) and increase number of sequence dataset, access global scale distribution of your favorite bug became more feasible. Basic after acquiring your sequence data either in the lab or through public databases, all you need is to determine which sequence belongs to which bug, like sorting the sequences into boxes where each represents different individuals. The changeling is to determine the individual units and their close relatives, so you don’t end up sorting the sequences into the wrong box. 

One language we all use in biology to define the limits between individuals is the one provided by taxonomy where the individuals are categorized into species, the different species into genera, etc. Often the combination of taxonomy and metabarcoding method brings interesting insights into marine phytoplankton distribution and ecology. 

Below examples of how we combined the taxonomic knowledge with metabarcoding data to unveil the distribution of phytoplankton groups. 


## Bolidophyceae, the sister group of diatoms
Naked flagellated cells (previously called Bolidomonas, A) was initially described as belonging to the class Bolidophyceae. With the isolation of one silicified member of Parmales group (B), Bolidophyceae and Parmales were thought to be sister classes of Phytoplankton.  In our study ([Ichinomiya and Lopes dos Santos et al., 2016](files/papers/ Ichinomiya_2016.pdf)), we started by acquiring sequence data from strains from both groups to analyze the evolutionary relationships among the several silicified and flagellated cells isolated from different oceanic regions. We were able to show that the silica covered cells actually belong to the same Class (Bolidophyceae) of the naked flagellated cells indicating that Bolidophyceae may have a life cycle with motile flagellated and non-motile silicified stages. We then assessed the distribution of the major Bolidophyceae taxa across the world oceans by using metabarcoding data set obtained in the frame of the Tara Oceans expedition. We observed that the silicified cells are mostly found in cold-water while the naked flagellate have temperate and tropical distributions. 
Many questions about Bolidophyceae remain open and in collaboration with [Akira Kuwata]( https://www.researchgate.net/profile/Akira_Kuwata) we plan to continue exploring the diversity of this groups and the ecological factors that determine its growth and differentiation into flagellate and silicified cells.  
![](/img/Bolido1.png)

## Chloropicophyceae also known as Prasinophytes clade VII  
Chloropicophyceae is a group of pico (2 – 3 µm) planktonic green algae that were known as prasinophytes clade VII ([Lopes dos Santos et al., 2017b ](files/papers/ Lopes_2017b.pdf)). 
Since the isolation of the first strain in 1965, numerous rRNA sequences belonging to this group have been retrieved from the marine environment as well as many strains have been isolated. We gathered the environmental sequence data available from strains and public dataset to access the diversity these tiny little green balls. First, by using different gene markers, we demonstrated that this group is composed of at least 10 different clades: A1 to A7 and B1 to B3 ([Lopes dos Santos et al., 2017a](files/papers/ Lopes_2017a.pdf)) which today is represented by at least one described species.  By using metabarcoding data from Tara Oceans and Ocean sampling day projects we were able to show that this group is one of the dominant Chlorophyta (green algae) group in oceanic waters, especially in areas where nutrients concentrations are moderately low.
Currently, in collaboration with [Claude Lemieux] (https://www.researchgate.net/profile/Claude_Lemieux2) from Laval University we are gathering information from Chloroplast genomes of Chloropicophyceae species to access the relationship among them and possible environment adaptions. 
![](/img/chloropico.png)




